applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py

```diff
@@ -10,32 +27,62 @@ def CreateSolver(main_model_part, custom_settings):
     if (type(custom_settings) != KratosMultiphysics.Parameters):
         raise Exception("input is expected to be provided as a Kratos Parameters object")
 
+    CheckForParallelism(custom_settings)
+    CheckForSolverType(custom_settings)
+
     parallelism = custom_settings["problem_data"]["parallel_type"].GetString()
     solver_type = custom_settings["solver_settings"]["solver_type"].GetString()
 
     # Solvers for OpenMP parallelism
     if (parallelism == "OpenMP"):
-        if (solver_type == "Implicit"):
-            solver_module_name = "structural_mechanics_implicit_dynamic_solver"
+        if (solver_type == "Dynamic"):
+            CheckForAnalysisType(custom_settings)
```

Comment:
Because this pr will break existing cases by changing the json format, I would like to push that we also clean up some inconsistencies with CamelCase versus lower_case_with_underscores. From the wiki page style guide: "The name of the property should be written in lowercase with underscores "_" in between words if necessary". Unless I am not understanding something about the convention, can you make Dynamic, Static, EigenValue, etc. lower case with underscores?

Reply:
fixed